api/Element.json

```diff
@@ -8378,54 +8378,6 @@
           }
         }
       },
-      "tabStop": {
```

Comment:
I can find no trace of this in Chromium, Gecko or WebKit so this removal seems correct.